README.md

```diff
@@ -5,32 +5,34 @@ ATM - Auto Tune Models
 [![Coverage status](https://codecov.io/gh/HDI-project/ATM/branch/master/graph/badge.svg)](https://codecov.io/gh/HDI-project/ATM)
 [![Documentation](https://readthedocs.org/projects/atm/badge/?version=latest)](http://atm.readthedocs.io/en/latest/)
 
-ATM is an open source software library under ["The human data interaction project"](https://hdi-dai.lids.mit.edu/) at MIT. It is a distributed, scalable AutoML system designed with ease of use in mind. 
+ATM is an open source software library under the [*Human Data Interaction* project](https://hdi-dai.lids.mit.edu/) (HDI) at MIT. It is a distributed, scalable AutoML system designed with ease of use in mind.
```

Comment:
This is outside the scope of the pull request, but shouldn't it be "Human-Data Interaction" in general? As is, it reads to me like the project's mission is "interacting with data about humans" rather than "helping humans interact with data."

Reply:
That's a reasonable interpretation, I'm just going off the website 😛